### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by kjhman21**
## Proposed changes

To go to the QA process, execute release_rc.sh.
It creates a tag like `v1.2.0-rc.1`, publishes packages, and creates a PR
for the QA process.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: release_rc.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Fetch tags from upstream first
+git fetch -t git@github.com:klaytn/klaytn.git
+
+VERSION=$(go run build/rpm/main.go version)
+echo "Trying to tag v$VERSION"
```
Comment (aidan-kwon) [inline release_rc.sh]:
```suggestion
echo "Trying to tag $VERSION"
```
The tag already has "v"

File: release_rc.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  exit 1
+fi
+
+PREV_RCVERSION_FULL=`git tag | grep "$VERSION" | sort | tail -1`
+RCSUFFIX="1"
+if [ ! -z "$PREV_RCVERSION_FULL" ]; then
+  RCSUFFIX=$(echo "${PREV_RCVERSION_FULL##*.}"+1 | bc)
+fi
+
+RCVERSION="v$VERSION-rc.$RCSUFFIX"
```
Comment (aidan-kwon) [inline release_rc.sh]:
```suggestion
RCVERSION="$VERSION-rc.$RCSUFFIX"
```

File: release_rc.sh (hunk 1 above)
Comment (kjhman21) [reply release_rc.sh]: nice catch! thanks!

File: release_rc.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  exit 1
+fi
+
+PREV_RCVERSION_FULL=`git tag | grep "$VERSION" | sort | tail -1`
+RCSUFFIX="1"
+if [ ! -z "$PREV_RCVERSION_FULL" ]; then
+  RCSUFFIX=$(echo "${PREV_RCVERSION_FULL##*.}"+1 | bc)
+fi
+
+RCVERSION="$VERSION-rc.$RCSUFFIX"
```
Comment (aidan-kwon) [inline release_rc.sh]: What if `$VERSION` was an rc format like "v1.0.0-rc.1"?
Than, `RCVERSION` will be "v1.0.0-rc.1-rc.2".

Comment (kjhman21) [reply release_rc.sh]: $VERSION always be like `v1.2.0`. `-rc.2` will be attached during CI process.

**Pull request merged**